network-api/networkapi/wagtailpages/pagemodels/blog/blog_category.py

```diff
@@ -51,16 +52,20 @@ def get_categories():
         """
         WARNING: this function is referenced by two migrations:
 
-        - mozfest/0014_auto_20200406_2109.py
-        - wagtailpages/0095_auto_20200406_2109.py
+        - mozfest/0001_10_0015.py
+        - wagtailpages/0001_initial.py
 
         This means that renaming/(re)moving this function will require
         back-updating those two migrations, as "from scratch" migrations
         (compared to update-only migrations) will throw errors when trying
         to apply this function from its original location.
         """
-        choices = [(cat.name, cat.name) for cat in BlogPageCategory.objects.all()]
-        choices.sort(key=lambda c: c[1])
+        choices = []
+        try:
+            choices = [(cat.name, cat.name) for cat in BlogPageCategory.objects.all()]
+            choices.sort(key=lambda c: c[1])
+        except ProgrammingError:
+            pass
```

Comment:
Let's add some comments to explain to future readers why this exception is here.